docker-compose.yml

```
@@ -12,7 +12,10 @@ services:
         target: /run/host-services/ssh-auth.sock
     environment:
      - SSH_AUTH_SOCK=/run/host-services/ssh-auth.sock
-    command: ["ruby", "run_dark_blue.rb"]
+    depends_on:
+      database:
+        condition: "service_healthy"
+    command: ["sh", "-c","ruby run_dark_blue.rb && tail -f /dev/null"] 
```

What's the thinking here for running both of these? So you can do things afterward? I guess I'm personally more likely to want one or the other.

---

What do you think about changing this to the following? With a fresh database volume, `docker compose up` fails because migrations aren't applied yet. This should fix that, and it's harmless to run `rake db:migrate` repeatedly.

```suggestion
    command: ["sh", "-c", "rake db:migrate && ruby run_dark_blue.rb"] 
```
We can still use this approach if we want to do it all separately.
```
docker compose up database
docker compose run dark-blue bash
rake db:migrate
ruby run_dark_blue.rb
```

---

Whatever we end up doing here, let's make sure the instructions in the Docker section of the `README.md` are up to date.

---

> What's the thinking here for running both of these? So you can do things afterward? I guess I'm personally more likely to want one or the other.

I added the later command because the dark blue gets exited with code 0 and its works only if you explicitly keep a foreground process running.